diff --git a/src/Utils/Utils/Scf/OrbitalPerturbation/UniqueRandomNumbersGenerator.h b/src/Utils/Utils/Scf/OrbitalPerturbation/UniqueRandomNumbersGenerator.h
index 04cddc3..3f10aa5 100644
--- a/src/Utils/Utils/Scf/OrbitalPerturbation/UniqueRandomNumbersGenerator.h
+++ b/src/Utils/Utils/Scf/OrbitalPerturbation/UniqueRandomNumbersGenerator.h
@@ -12,6 +12,7 @@
 #include <cassert>
 #include <numeric>
 #include <vector>
+#include <random>
 
 namespace Scine {
 namespace Utils {
@@ -62,7 +63,11 @@ std::vector<IntegerType> UniqueRandomNumbersGenerator<IntegerType>::generate(uns
 
   std::vector<IntegerType> shuffler(max_ - min_ + 1);
   std::iota(shuffler.begin(), shuffler.end(), min_);
-  std::random_shuffle(shuffler.begin(), shuffler.end());
+
+  std::random_device rd;
+  std::mt19937 g(rd());
+  std::shuffle(shuffler.begin(), shuffler.end(), g);
+
   return {shuffler.begin(), shuffler.begin() + N};
 }
 
